html/sites.php

```diff
@@ -2,8 +2,19 @@
 
 require_once __DIR__.'/../vendor/autoload.php';
 
-$resultsStorage = new \ScraperBot\Storage\SqlLite3Storage('../railerdb.sqlite3');
-$crawls = $resultsStorage->getTimeStamps();
+$resultsStorage = new \ScraperBot\Storage\SqlLite3Storage('../glitcherbot.sqlite3');
+
+$compare = [];
+if (isset($_GET['date1']) && $_GET['date2']) {
+    echo $_GET['date1'] . ' date2: ' . $_GET['date2'];
+    $compare['date1'] = $_GET['date1'];
+    $compare['date2'] = $_GET['date2'];
+}
+
+if (isset($_GET['latest'])) {
```

Comment:
I'm not sure what the behaviour is for `onlyLatest`?

Reply:
show only latest results, but I'll get rid of that for now.